bot/exts/backend/error_handler.py

```diff
@@ -98,6 +106,42 @@ async def on_command_error(self, ctx: Context, e: errors.CommandError) -> None:
             # ExtensionError
             await self.handle_unexpected_error(ctx, e)
 
+    @Cog.listener("on_app_command_error")
+    async def on_app_command_error(self, interaction: Interaction, e: app_commands.AppCommandError) -> None:
+        """
+        Error handler for app commands.
+
+        See on_command_error for detailed description.
+        """
+        await interaction.response.defer(ephemeral=True)
+
+        debug_message = (
+            f"Command {interaction.command} invoked by {interaction.user} with error "
```

Comment:
Use `command.qualified_name` here as well

Reply:
Updated in https://github.com/python-discord/bot/pull/2446/commits/13aa29a6ed1069660b7d20aa15222d953a8bf59b